testsuite/forge/src/backend/k8s/chaos/network_bandwidth.yaml

```diff
@@ -0,0 +1,15 @@
+kind: NetworkChaos
+apiVersion: chaos-mesh.org/v1alpha1
+metadata:
+  namespace: {namespace}
+  name: forge-namespace-{rate}mbps-bandwidth
```

Comment:
Do we need a `label` selector for this as well?

Reply:
ah yea we do -- good catch!